### DISCUSSION TITLE
Getting “ModuleNotFoundError” even though it just got installed by pip

### DISCUSSION THREAD

**Discussion by yishans** [MeVisLab/pythonqt#253] [Q&A]
1. I had intall a new module successful,   websockets  for example.
2. But in pythonqt,  i cant import the module, error ocurred "ModuleNotFoundError".

how to use a new 3td module in pythonqt?

thank you!

**Comment by mrbean-bremen** [comment]
I guess you mean Qt3D. This is (currently) not included (like a number of other optional modules), so it can't be added out of the box.
You would need to create a typesystem file for the module (see the *typesystem_xxx.xml* files in the generator subdirectory), add it to the build scripts (*generator.qrc* and *build_all.txt*) and probably more.
